modules/aws/rds/main.tf

```diff
@@ -62,7 +63,8 @@ module "db" {
 
   subnet_ids = var.vpc.database_subnets
   vpc_security_group_ids = [module.security_group.security_group_id]
-
+  create_db_subnet_group = var.create_db_subnet_group
+  create_random_password = var.create_random_password
```

Comment:
what happens if `create_random_password` is set to `true` and changes are applied during the update flow?
has this been tested?

Reply:
@suryaoruganti in our current ui, we are always asking user for username and password. Hence we will not be populating `create_random_password`  and it will always default to `false`.